src/devices/steelseries_arctis_1.c

```diff
@@ -112,7 +112,7 @@ static int arctis_1_send_inactive_time(hid_device* device_handle, uint8_t num)
     // the range of the Arctis 7 seems to be from 0 to 0x5A (90)
     // num = map(num, 0, 128, 0x00, 0x5A);
 
-    uint8_t data[31] = { 0x06, 0x51, num };
+    uint8_t data[31] = { 0x06, 0x53, num };
```

Comment:
Im not 100% sure about this change. We may need someone else with another headset to test if it still works for them. In doubt we need to put it inside a different device file.

Reply:
I have an Arctis 1 Wireless, and can confirm these settings are correct. The inactivity shutdown timer is working now

`Bus 003 Device 007: ID 1038:12b3 SteelSeries ApS SteelSeries Arctis 1 Wireless`

Reply:
Maybe they were not correct from the beginning